cmd/get/bom.go

```diff
@@ -35,7 +35,7 @@ var getBomInfoByMacAddress = &cobra.Command{
 		}
 
 		if flagsGetBomByMacAddress.hasAOCMacAddr() && flagsGetBomByMacAddress.hasBMCMacAddr() {
```

Comment:
With cobra marking these as mutually exclusive, I don't think we'll ever get here. I think the library will terminate on an error.